eq_cir_proxy_service/services/instrument/instrument_conversion_service.py

```diff
@@ -0,0 +1,113 @@
+                "message": f"Invalid {source} version: {version}",
+            },
+        ) from e
+
+
+async def convert_instrument(instrument: Instrument, target_version: str) -> Instrument:
+    """Requests conversion of the instrument from Converter Service.
+
+    Parameters:
+    - instrument: The instrument.
+    - current_version: The current version of the instrument.
+    - target_version: The target version of the instrument.
+
+    Returns:
+    - dict: The converted instrument.
+    """
+    if not instrument.get("validator_version"):
+        logger.error("Instrument version is missing")
+        raise HTTPException(
+            status_code=status.HTTP_404_NOT_FOUND,
+            detail={"status": "error", "message": exception_messages.EXCEPTION_400_INVALID_INSTRUMENT},
+        )
+
+    current_version = str(instrument["validator_version"])
+    parsed_current_version = safe_parse("current", current_version)
+    parsed_target_version = safe_parse("target", target_version)
+
+    if parsed_current_version < parsed_target_version:
+        logger.info(
+            "Instrument requires updating. Requesting conversion of instrument by converter service.",
+            current_version=current_version,
+            target_version=target_version,
+        )
+
+        converter_service_base_url = os.getenv("CONVERTER_SERVICE_API_BASE_URL")
+        converter_service_endpoint = os.getenv("CONVERTER_SERVICE_CONVERT_CI_ENDPOINT", "/schema")
+
+        if not converter_service_base_url:
+            logger.error("CONVERTER_SERVICE_API_BASE_URL is not configured.")
+            raise HTTPException(
+                status_code=500,
+                detail={
+                    "status": "error",
+                    "message": "CONVERTER_SERVICE_API_BASE_URL configuration is missing.",
+                },
+            )
+        if not converter_service_endpoint:
+            logger.error("CONVERTER_SERVICE_CONVERT_CI_ENDPOINT is not configured.")
+            raise HTTPException(
+                status_code=500,
+                detail={
+                    "status": "error",
+                    "message": "CONVERTER_SERVICE_CONVERT_CI_ENDPOINT configuration is missing.",
+                },
+            )
+
+        url = f"{converter_service_base_url}{converter_service_endpoint}"
+        try:
+            async with AsyncClient() as client:
+                response = await client.post(
+                    url,
+                    json={"instrument": instrument},
+                    params={"current_version": current_version, "target_version": target_version},
+                )
+        except RequestError as e:
+            logger.exception("Error occurred while converting instrument: ", error=e)
+            raise HTTPException(
+                status_code=500,
+                detail={
+                    "status": "error",
+                    "message": "Error connecting to Converter service.",
```

Comment:
```suggestion
                    "message": "Error connecting to Converter Service.",
```
Replaces the error message to use the name "Converter Service". Other instances can also be updated.